crml/nft/src/tests.rs

```diff
@@ -1071,7 +1071,7 @@ fn auction_bundle() {
 		let _ = <Test as Config>::MultiCurrency::deposit_creating(&buyer, PAYMENT_ASSET, 1_000);
 		assert_ok!(Nft::bid(Some(buyer).into(), listing_id, 1_000));
 		// end auction
-		let _ = Nft::on_initialize(System::block_number() + 1);
+		let _ = Nft::on_initialize(System::block_number() + 40);
```

Comment:
could you use the constant here instead of `40` to more clearly show it's moving past the auto extended period